features/support/paths.rb

```diff
@@ -17,26 +17,31 @@ def path_to(page_name)
       '/'
 
     when /^the new vendor page$/
-      new_vendor_path
+      new_admin_vendor_path
 
     when /^the vendors page$/
```

Comment:
The naming of this and the vendors index page seems a bit confusing. Changing to something like "admin vendors page" or "admin-facing vendors page" is probably better. Something similar should be done for the customer-facing index pages as well.